servers/src/main/java/tachyon/worker/block/evictor/LRFUEvictor.java

```diff
@@ -111,6 +111,9 @@ public LRFUEvictor(BlockMetadataManagerView view, Allocator allocator) {
    */
   private StorageDirView cascadingEvict(long bytesToBeAvailable, BlockStoreLocation location,
       EvictionPlan plan, List<Map.Entry<Long, Double>> sortedCRF) {
+    Preconditions.checkNotNull(location);
+    Preconditions.checkNotNull(plan);
+    Preconditions.checkNotNull(sortedCRF);
```

Comment:
+1 for calvin's comment. I felt it is not necessary for internal methods to check the input.